cmip6_downscaling/data/observations.py

```diff
@@ -34,112 +32,76 @@ def open_era5(var):
     Returns
     -------
     xarray.Dataset
-        An hourly dataset for one variable.
-    """
-    print("getting stores")
-    col = intake.open_esm_datastore(
-        "https://cmip6downscaling.blob.core.windows.net/cmip6/ERA5_catalog.json"
-    )
-    stores = col.df.zstore
-    era5_var = variable_name_dict[var]
-    store_list = stores[stores.str.contains(era5_var)].to_list()
-    ds = xr.open_mfdataset(
-        store_list,
-        engine="zarr",  # these options set the inputs and how to read them
-        consolidated=True,
-        parallel=True,  # these options speed up the reading of individual datasets (before they are combined)
-        combine="by_coords",  # these options tell xarray how to combine the data
-        # data_vars=['air_temperature_at_2_metres_1hour_Maximum']  # these options limit the amount of data that is read to only variables of interest
-    ).drop("time1_bounds")
-    print("return mfdataset")
-    return ds
-
-
-def load_obs(obs_id, variable, time_period, domain, agg_func=np.max):
+        A daily dataset for one variable.
     """
+    stores = [
+        f'https://cmip6downscaling.blob.core.windows.net/cmip6/ERA5_daily/{y}'
+        for y in range(1979, 2021)
+    ]
+    ds = xr.open_mfdataset(stores, engine='zarr', consolidated=True)
+    return ds[[var]]
+
+
+def load_obs(obs_id: str, variable: str, time_period: slice) -> xr.Dataset:
+    """Load a temporal subset of an observational dataset for one variable
     Parameters
     ----------
-    obs_id : [type]
-        [description]
-    variable : [type]
-        [description]
-    time_period : [type]
-        [description]
-    domain : [type]
-        [description]
+    obs_id : str
+        name of observation dataset. currently only "ERA5" is supported
+    variable : str
+        name of variable following CMIP conventions (e.g. tasmax, tasmin, pr)
+    time_period : slice
+        time period you want to subset. e.g. slice('1985', '2015'). using
+        full years is recommended if you are going to be doing the spatial
+        anomaly calculations.
 
     Returns
     -------
-    [xarray dataset]
-        [Chunked {time:-1,lat=10,lon=10}]
+    open_era5(variable).sel(time=time_period) : xr.Dataset
```

Comment:
I think the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#returns) way to reference an unnamed returned object is:
```suggestion
    xr.Dataset
```